src/sass/partials/layouts/_content-card.scss

```diff
@@ -0,0 +1,35 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  margin: 2rem 0 0;
+  width: 100%;
+
+
+  @include breakpoint(setting-get('two grid cards')) {
+    margin-right: 4%;
+    width: 48%;
```

Comment:
Where does this 48% come from? Is this suppose to be 50% with a 2% margin?. Width should be 50% and there should be padding in the inside at a fixed width for the gutter